### PR TITLE
GIVCAMP-72 | Split poster component; Storyblok SbCta; Container update

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- `SplitPoster` component with support for multiple variants
- `ImageOverlay` component
- Remove `Section` component and absorb its functionalities into `Container`, add support for vertical paddings
- Remove starter `Feature` component

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to the preview and look at the poster variants
Screenshot 2023-04-19 at 2 24 07 PM
Screenshot 2023-04-19 at 2 23 59 PM
Screenshot 2023-04-19 at 2 23 51 PM
Screenshot 2023-04-19 at 2 23 42 PM

2. Play with the Split Poster component in Storyblok
3. Look at code

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-72

### CODE REVIEW

File: src/tailwind/plugins/theme/gc-screens.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * 2 extra breakpoints + print support
+ * 3xl: 2xl + 2 x the screen edge margins
+ * 4xl: 2000px
+ */
+
+module.exports = function () {
+  return {
+    '3xl': '1700px',
+    '4xl': '2000px',
```
Comment (sherakama) [inline src/tailwind/plugins/theme/gc-screens.js:10]: I like big screens

Comment (yvonnetangsu) [reply src/tailwind/plugins/theme/gc-screens.js:10]: Me too 😄 We have some elements that are touching the edge only up to a certain reasonable point, thus these.

File: src/components/Storyblok/SbCta.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      sbLink={link}
+      variant={ctaVariant}
+      color={color}
+      srText={srText}
+      icon={icon}
+    >
+      {label}
+    </CtaLink>
+  );
+};
```
Comment (yvonnetangsu) [inline src/components/Storyblok/SbCta.tsx:46]: Add the Storyblok Cta `SbCta` that connects to our `CtaLink` components

File: src/pages/index.tsx [hunk 3]
```diff
@@ -368,19 +368,19 @@ const IndexPage = ({ data }) => {
             Digital Red with really long label
           </CtaLink>
         </Grid>
-      </Section>
-      <Section>
+      </Container>
+      <Container py={9} bgColor="black">
```
Comment (yvonnetangsu) [inline src/pages/index.tsx:372]: Can add vertical padding to Containers this way now

File: src/utilities/getProcessedImage.ts [hunk 4]
```diff
@@ -30,17 +34,19 @@ export const getProcessedImage = (
...
-    if (focus) {
-      myFilters += `:focal(${focus})`;
-    } else {
-      myParams += '/smart';
+    if (width !== '0' && height !== '0') {
+      if (focus) {
+        myFilters += `:focal(${focus})`;
+      } else {
+        myParams += '/smart';
+      }
```
Comment (yvonnetangsu) [inline src/utilities/getProcessedImage.ts:49]: Fixup for image processing. Previously it didn't actually add the crop if either the width or the height is set to 0

**Comment by yvonnetangsu** [comment]
Going to merge this as Kerri has given the initial ok 🤠  Will finetune later.

**Pull request merged**